src/modules/notifications.ts

```diff
@@ -56,6 +60,22 @@ export class NotificationsAPI extends BaseClient {
       headers: { 'Content-Type': 'application/json' }
     })
   }
+
+  async putEmail(email: string) {
+    return this.fetch<Subscription>('/set-email', {
```

Comment:
[Neither of these two new fetches return a subscription](https://github.com/decentraland/notifications-workers/blob/7ee172c73cd456fbedd9175cac57db424091a1af/inbox/src/controllers/handlers/unconfirmed-email-handlers.ts#L33). Would you mind changing it?